feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -176,6 +179,7 @@ internal fun ForYouScreen(
         newsFeed(
             feedState = feedState,
             onNewsResourcesCheckedChanged = onNewsResourcesCheckedChanged,
+            onTopicClick = onTopicClick,
```

Comment:
This was on purpose (though I did not go out of my way to add it everywhere in every file I touched). Look at all the lines that this PR touched just because they didn't have trailing commas!

@dturner to chime in on whether there's a convention? 😄

Reply:
Following this up in another PR.